src/System.Numerics.Vectors/src/System/Numerics/Vector3.cs

```diff
@@ -43,13 +42,7 @@ public partial struct Vector3 : IEquatable<Vector3>, IFormattable
         /// Returns the hash code for this instance.
         /// </summary>
         /// <returns>The hash code.</returns>
-        public override int GetHashCode()
-        {
-            int hash = this.X.GetHashCode();
-            hash = HashHelpers.Combine(hash, this.Y.GetHashCode());
-            hash = HashHelpers.Combine(hash, this.Z.GetHashCode());
-            return hash;
-        }
+        public override int GetHashCode() => HashCode.Combine(X, Y, Z);
```

Comment:
(nit) Vector2, 3, and 4 all use different styles here.  Vector3 has the Combine(X, Y, Z) all on one line.  2 and 4 split it on multiple lines.  IMO - putting them all on a single line reads easier.